examples/ex_hpiFit/main.cpp

```diff
@@ -156,17 +156,18 @@ int main(int argc, char *argv[])
     fiff_int_t quantum = ceil(quantum_sec*pFiffInfo->sfreq);
 
     // create time vector that specifies when to fit
-//    int N = ceil((last-first)/quantum);
-//    RowVectorXf time(N);
-//    for(int i = 0; i < N; i++){
-//        time(i) = i * dT_sec;
-//    }
-
-    // Read Quaternion File
-    Eigen::MatrixXd pos;
-    qInfo() << "Specify the path to your position file (.txt)";
-    UTILSLIB::IOUtils::read_eigen_matrix(pos, QCoreApplication::applicationDirPath() + "/mne-cpp-test-data/Result/ref_hpiFit_pos.txt");
-    RowVectorXd time = pos.col(0);
+    int N = ceil((last-first)/quantum);
+    RowVectorXf time(N);
+    for(int i = 0; i < N; i++){
+        time(i) = i * dT_sec;
+    }
```

Comment:
You could try using `RowVectorXf::LinSpaced(N, 0, N-1) * dT_sec` for this, see Eigen docu [here](https://eigen.tuxfamily.org/dox/group__TutorialAdvancedInitialization.html).